index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capstone Module1</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.html">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="./Images/logo-no-background.svg" type="image/svg" />
+  </head>
+  <body>
+    <section class="nav-bar">
+      <div id="ham-menu">
+        <img
+          onclick="clickX()"
+          id="x"
+          src="https://img.icons8.com/sf-regular/48/FFFFFF/multiply.png"
+        />
+        <ul class="ham">
+          <li>
+            <a href="#" class="ham-items" onclick="clickX()"
+              >Home</a
+            >
+          </li>
+          <li>
+            <a href="about.html" class="ham-items" onclick="clickX()">About</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Program</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Join</a>
+          </li>
+          <li>
+            <a href="#contact" class="ham-items" onclick="clickX()">Sponsor</a>
+          </li>
+        </ul>
+      </div>
+      <header class="head flex-row">
+        <nav>
+          <img
+            onclick="clickMenu()"
+            id="menu"
+            src="./Images/Icon - Menu.png"
+            alt="Menu Logo"
+            class="menu-logo"
+          />
+          <a href="">
+            <img class="logo" src="Images/logo-black.png" alt="logo" />
+          </a>
+          <ul class="menu_items flex-row">
```

Comment:
- [ ] please kindly make your mobile menu responsive till it reaches the desktop version
- [ ] please ensure that it disappears when it reaches desktop version
![Screenshot (971)](https://user-images.githubusercontent.com/65298048/233433172-f872e7e3-8fd6-47e0-97eb-239f899b41f8.png)
![Screenshot (974)](https://user-images.githubusercontent.com/65298048/233433229-a2e42e13-d592-4865-b4a8-903cd845324d.png)